seungmi/week4/큰수만들기.py

```diff
@@ -0,0 +1,17 @@
+def solution(number, k):
+    answer = []
+    #stack을 이용해 앞자리 부터 검사, 단 k개의 숫자를 제거하면 이후 숫자는 그냥 붙임
+    for num in number:
+        if not answer:
+            answer.append(num)
+            continue
+        if k > 0:
+            while answer[-1] < num:
+                answer.pop()
+                k -= 1
+                if not answer or k <= 0:
+                    break
```

Comment:
if문 조건들을 while문 조건으로 한번에 넣어도 될 것 같아요
```suggestion
        while len(answer) > 0 and answer[-1] < num and k > 0:
            k -= 1
            answer.pop()
```